cognite/neat/rules/models/_rules/dms_architect_rules.py

```diff
@@ -125,46 +127,60 @@ class DMSProperty(SheetEntity):
     container_property: str | None = Field(None, alias="ContainerProperty")
     view: ViewType | None = Field(None, alias="View")
     view_property: str | None = Field(None, alias="ViewProperty")
-    index: str | None = Field(None, alias="Index")
-    constraint: str | None = Field(None, alias="Constraint")
+    index: StrListType | None = Field(None, alias="Index")
+    constraint: StrListType | None = Field(None, alias="Constraint")
+
+    @field_validator("value_type", mode="before")
```

Comment:
I guess this could not fit under `_types.py` ?

Reply:
It is only used here, which is why I do not put it into the types.